src/api/controllers/messages.ts

```diff
@@ -94,13 +94,6 @@ export async function handleInternalMessageUpdate(data: DiscordPayload) {
     pinned: cachedMessage.pinned,
   };
 
-  // Messages with embeds can trigger update but they wont have edited_timestamp
-  if (
-    !payload.edited_timestamp ||
-    (cachedMessage.content !== payload.content)
-  ) {
-    return;
-  }
-
+  if (!payload.edited_timestamp) return;
```

Comment:
I am not sure this really should change. I'm not really convinced. IMO for 99% of bots this is the ideal version of this event. If a bot dev needs more its easily changeable with a controller.

Reply:
It also help prevent wasting CPU usage for all these bots that don't need it.

Reply:
Having the original logic would not fire messageUpdate at all